app/screens/channel/channel.js

```diff
@@ -141,6 +150,25 @@ class Channel extends PureComponent {
         }
     });
 
+    handleAppStateChange = async (appState) => {
+        const {actions} = this.props;
+        const {
+            closeWebSocket,
+            initWebSocket,
+            startPeriodicStatusUpdates,
+            stopPeriodicStatusUpdates,
+        } = actions;
+        const isActive = appState === 'active';
+
+        if (isActive) {
+            initWebSocket(Platform.OS);
```

Comment:
We don't need to track if the websocket has already been initialized, do we? The code is good to merge as long as we don't think that's the case

Reply:
great question, let me check